app/code/community/Bolt/Boltpay/Model/FeatureSwitch.php

```diff
@@ -48,4 +58,15 @@ public function updateSwitchesFromBolt()
         Mage::getModel('core/config')->saveConfig('payment/boltpay/featureSwitches', json_encode($switches));
         Mage::getModel('core/config')->cleanCache();
     }
+
+    public function updateSwitchesFromBoltIfNeeded()
+    {
+        if ($this->needUpdateFeatures) {
+            $this->updateSwitchesFromBolt();
+        }
+    }
+
+    public function needUpdateFeatures() {
```

Comment:
Doesn't seem like this is used anywhere?

Reply:
Sorry, Aden, I didn't include one file into PR. Updated.